official/resnet/keras/README.md

```diff
@@ -60,16 +60,35 @@ There are more flag options you can specify. Here are some examples:
 data, are used;
 - `--batch_size`: the batch size used for the model;
 - `--model_dir`: the directory to save the model checkpoint;
-- `--train_epochs`: number of epoches to run for trianing the model;
+- `--train_epochs`: number of epoches to run for training the model;
 - `--train_steps`: number of steps to run for training the model. We now only
-support a number that is smalle than the number of bathces in an epoch.
-- `--skip_eval`: when set to true, evaluation of the model will be skipped. 
+support a number that is smaller than the number of batches in an epoch.
+- `--skip_eval`: when set to true, evaluation as well as validation during 
+training is skipped
+
+For example, this is a typical command line to run with ImageNet data with 
+batch size 128 per GPU:
+
+```bash
+python -m keras_imagenet_main \
+--data_dir=/data/imagenet/ \
+--model_dir=/tmp/model_dir/something \
+--hooks ExamplesPerSecondHook \
+--num_gpus=2 \
+--batch_size=64 \
+--resnet_size=50 \
```

Comment:
is this argument even respected in the keras model? it didn't use to be

Reply:
Deleted.